### PR TITLE
Update gravatar hash tool

### PR DESCRIPTION
**Pull request by DigTK**
Changes the tool you use to get your gravatar hash from a third party tool to one from gravatar directly.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ And access the application at http://localhost:4000/
 Member data is located under `_collections/members`. Copy an existing profile, name it something unique (preferably using your username on the devanooga Slack) and fill it out using these properties:
 
 - **`name`** (required) Your full name as it should appear on the site.
-- **`gravatar`** (required) Your gravatar hash. [Use this tool to get your gravatar hash.](http://aruljohn.com/gravatar/) _For example, @StrangeWill’s result is `http://www.gravatar.com/avatar/6789f8bd72612e941f9a7ae6f414b2ea?s=48&d=identicon`; the hash is this part: `6789f8bd72612e941f9a7ae6f414b2ea`._
+- **`gravatar`** (required) Your gravatar hash. [Use this tool to get your gravatar hash](https://en.gravatar.com/site/check/), after you enter your email address it will be displayed as `Email Hash`. _For example, @StrangeWill’s result is `Email Hash: 6789f8bd72612e941f9a7ae6f414b2ea`._
```
Comment (agarzola) [inline readme.md]: The instructions are OK, but can be improved. I suggest this modified version:
>- **`gravatar`** (required) Your gravatar hash. [Use this tool to get your gravatar hash.](https://en.gravatar.com/site/check/) After you enter your email address and click the _Check_ button, the tool will display some information below the form. The value you need is the one labeled **Email Hash**. (For example, @StrangeWill’s email hash is `6789f8bd72612e941f9a7ae6f414b2ea`.)

While we’re on the subject, does it make sense to add information about what gravatar is and why we require it? I’d like to get @brb3 @StrangeWill’s input on this.

Comment (StrangeWill) [inline readme.md]: I guess a quick blurb about it is fine. I'd probably just say something like "this is what we use to display your picture on our page, click here for more details" or something, with a link to something on gravatar's site.

Comment (brb3) [inline readme.md]: How about
> `gravatar` (required) Your gravatar hash. Gravatar provides a single repository for your internet-wide avatar. We use this to display a picture of you on our member page. [Use this tool](https://en.gravatar.com/site/check/) to get your gravatar hash. After you enter your email address and click the Check button, the tool will display some information below the form. The value you need is the one labeled Email Hash. (For example, @StrangeWill’s email hash is `6789f8bd72612e941f9a7ae6f414b2ea`.)

Comment (agarzola) [inline readme.md]: That's really good, @brb3. I would italicize _Check_ (the _Check_ button) and use plural _members_ (on our members page).

**Pull request merged**